### PR TITLE
chore: add maintainers

### PR DESCRIPTION
**Pull request by jleach**
# Summary of Changes

Add maintainers file as per discussion on the last working group call.

# Related Issues

Please reference here any issue #'s that are relevant to this PR, or simply enter "N/A" if this PR does not relate to any existing issues.

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: MAINTAINERS.md
Comment (swcurran) [inline MAINTAINERS.md]: From what I can see in the Hyperledger Teams, this is not the right list. It should be:

| Name          | Github      | LFID |
| ------------- | ----------- | ---- |
| @jleach	Jason C. Leach | jleach | |
| @TelegramSam	Sam Curren | TelegramSam | |
| @cvarjao	Clécio Varjão | cvarjao | |
| @amanji	Akiff Manji | amanji | |
| @karimStekelenburg	Karim Stekelenburg | karimStekelenburg | |
| @TimoGlastra	Timo Glastra | TimoGlastra | |
| @JamesKEbert	James Ebert | JamesKEbert | |

See: https://github.com/orgs/hyperledger/teams?query=aries-mobile-agent

Comment (swcurran) [reply MAINTAINERS.md]: Not sure those are the right people -- just that those are the ones on the team.

@ryjones -- a general question. On the "committers" team, James is "Maintainer". However, I believe the definition that Hyperledger uses for Maintainer is really a "Committer" -- someone that can approve PRs and merge them.  Is that correct?

**Review by swcurran — CHANGES_REQUESTED** [review]
@jleach -- can you please update?  Then lets get this merged ASAP.  We have some other potential maintainers interested in joining them team.

I've heard through the grapevine that getting a PR merged in the repo is hard -- perhaps because the maintainers don't have enough time to do reviews?  Something to discuss at the next Aries Bifold meeting.

File: MAINTAINERS.md
Comment (jleach) [reply MAINTAINERS.md]: @swcurran On the last Bifold WG call @JamesKEbert proposed the base policy from which I went with based on the [MAINTAINTERS.md](https://github.com/hyperledger/anoncreds-spec/blob/main/MAINTAINERS.md) from the AnonCreds Spec repo. It outlined some guidelines for becoming and removing maintainers. When I created the list I roughly applied those guidelines for my decisions.

Based on that, Karim and Timo haven't contributed since 2021 so I'm hesitant to not follow the guidelines. I think following the guidelines is important so that nobody feels unfairly removed or added which can lead to conflict in a OSS / democratic process.

Comment (ryjones) [reply MAINTAINERS.md]: @swcurran the "maintainer" in that role means they can add people to that team in the GH UI

**Comment by swcurran** [comment]
IMHO — removing them would be a PR after creating the file. Agree with it or not, they are current maintainers.  I think the first cut at the file should reflect that.  Then a PR (with a discussion at the Aries Bifold Meeting) on changes, including removing them.  I thought the same thing when I saw their names — but didn’t go through the history.

**Comment by ryjones** [comment]
@swcurran I think one PR listing the people proposed to move to Emeritus would be OK.

**Comment by jleach** [comment]
@swcurran Agreed. We'll reflect reality then update.

File: MAINTAINERS.md
Comment (ryjones) [inline MAINTAINERS.md:9]:
```
| Akiff Manji        | amanji            |      |
| Clécio Varjão      | cvarjao           |      |
| James Ebert        | JamesKEbert       |      |
| Jason C. Leach     | jleach            |      |
| Karim Stekelenburg | karimStekelenburg |      |
| Sam Curren         | TelegramSam       |      |
| Timo Glastra       | TimoGlastra       |      |
```
sorted by github ID

File: MAINTAINERS.md
Comment (ryjones) [inline MAINTAINERS.md]:
```
| Akiff Manji        | amanji            |      |
| Clécio Varjão      | cvarjao           |      |
| James Ebert        | JamesKEbert       |      |
| Jason C. Leach     | jleach            |      |
| Karim Stekelenburg | karimStekelenburg |      |
| Sam Curren         | TelegramSam       |      |
| Timo Glastra       | TimoGlastra       |      |
```
sorted by ID

**Review by ryjones — CHANGES_REQUESTED** [review]
Please sort list by github ID

**Comment by jleach** [comment]
@ryjones Done.

**Review by ryjones — APPROVED** [review]

**Review by cvarjao — APPROVED** [review]

**Review by amanji — APPROVED** [review]

**Comment by ryjones** [comment]
@swcurran please approve

**Review by swcurran — APPROVED** [review]
Approving — but I’m not a maintainer, so probably shouldn’t be me…

:-)

**Comment by swcurran** [comment]
After all that — I just blindly approved.  We now need to remove the Community Specification License.  Since this is not a specification repository, it is not appropriate for it to be in the repo.  I’ll submit a PR.

**Pull request merged**
